ffi/derive/tests/ui_fail/derive_skip_field.stderr

```diff
@@ -1,17 +1,17 @@
 error[E0425]: cannot find function, tuple struct or tuple variant `FfiStruct__b` in this scope
```

Comment:
Errors in this file dont help to understand that `#[getset(skip)]` should not be used. Maybe this can be improved? At least in the other issue?

Reply:
this is not what this test is about. See [this comment](https://github.com/hyperledger/iroha/pull/2557#discussion_r934033768)